### PR TITLE
fix: hide language/text details on non-editor tabs

### PR DESCRIPTION
**Pull request by panekj**
- [ ] Added an entry to `CHANGELOG.md` if this change could be valuable to users

### CODE REVIEW

File: lapce-ui/src/status.rs [hunk 1]
```diff
@@ -482,6 +482,23 @@ impl Widget<LapceTabData> for LapceStatus {
...
+            // Do not draw file language and cursor/text details for non-file tabs
+            if let Some(editor_tab_id) = editor.tab_id {
+                if let Some(editor_tab) =
+                    data.main_split.editor_tabs.get(&editor_tab_id)
+                {
+                    if let Some(child) = editor_tab.active_child() {
+                        match child {
+                            EditorTabChild::Settings { .. }
+                            | EditorTabChild::Plugin { .. } => {
+                                return;
```
Comment (dzhou121) [inline lapce-ui/src/status.rs:494]: This could cause a surprise if we'll have something to paint after this.

Comment (dzhou121) [reply lapce-ui/src/status.rs:494]: I meant we should probably extract this paint in a new fn and return there.

Comment (panekj) [reply lapce-ui/src/status.rs]: @dzhou121 does that look better?

Comment (panekj) [reply lapce-ui/src/status.rs]: Might as well split document language name and details into own functions, but I'm lazy :P

**Pull request merged**